src/Illuminate/Broadcasting/Broadcasters/Ably/Utils.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace Illuminate\Broadcasting\Broadcasters\Ably;
+
+/**
+ * @author Sachin Shinde (sachinshinde7676@gmail.com)
+ */
+class Utils
+{
+    // JWT related PHP utility functions
+    /**
+     * @param  string  $jwt
+     * @return array
+     */
+    public static function parseJwt($jwt)
+    {
+        $tokenParts = explode('.', $jwt);
+        $header = json_decode(base64_decode($tokenParts[0]), true);
+        $payload = json_decode(base64_decode($tokenParts[1]), true);
+
+        return ['header' => $header, 'payload' => $payload];
+    }
+
+    /**
+     * @param  array  $headers
+     * @param  array  $payload
+     * @return string
+     */
+    public static function generateJwt($headers, $payload, $key)
```

Comment:
If there's parameters that can only be arrays then please add the type hints to the method signature. But do not add other types like string, bool, int, etc.

```suggestion
    public static function generateJwt(array $headers, array $payload, $key)
```